article/Experiment/GCF-ablation-study.tex

```diff
@@ -39,70 +39,70 @@ \subsection{Ablation study for GCF}\label{subsec:gcf-ablation-study}
+        GCF-sum              & 0.09724                       & 0.1988                                 & 0.03095                          & 0.06446             & 0.04075             & 0.07205             \\ \hline
+        GCF-sum-minus-sc     & 0.0956                        & 0.1962                                 & 0.03075                          & 0.0629              & 0.04114             & 0.07261             \\ \hline
+        GCF-sum-only-ip      & 0.09843                       & 0.199                                  & 0.02878                          & 0.06065             & 0.04114             & 0.07212             \\ \hline
     \end{tabular}
     \caption{NDCG and Recall of the changed methods.}
     \label{tab:ablation-results}
 \end{table*}
-\begin{figure}[h!]
-    \includegraphics[width=\linewidth]{figures/gcf-sum-ndcg.png}
-    \caption{NDCG@50 for the compared methods that utilize summation as layer combination on the Yelp2020 dataset.}
-    \label{fig:GCF-sum-NDCG-ablation-study}
-\end{figure}
-\begin{figure}[h!]
-    \includegraphics[width=\linewidth]{figures/amazon-cell-sport-gcf-sum-ndcg.png}
-    \caption{NDCG@50 for the compared methods that utilize summation as layer combination on the Amazon-Cell-Sport dataset.}
-    \label{fig:GCF-sum-NDCG-ablation-study-Amazon-Cell-Sport}
-\end{figure}
-Looking at \autoref{fig:GCF-sum-NDCG-ablation-study} it can be seen that LightGCN performs better than the other GCF methods that utilize weighted summation as layer combination.
-However on figure \autoref{fig:GCF-sum-NDCG-ablation-study-Amazon-Cell-Sport} it can be seen that the results vary a lot less, beside for \textit{gcf-sum-only-ip} that is clearly the worst performing.
 
-\begin{figure}[h!]
-    \includegraphics[width=\linewidth]{figures/gcf-ndcg-concat.png}
-    \caption{NDCG@50 for the compared methods that utilize concatenation as layer combination on the Yelp2020 dataset.}
-    \label{fig:GCF-NDCG-concat-ablation-study}
+\subsubsection{Concatenation and weighted summation}
+Looking at Yelp2020 and Amazon-Book on \autoref{tab:ablation-results} the methods that utilize concatenation as layer combination generally performs worse than the methods that utilize summation.
+On \autoref{fig:GCF-NDCG-ablation-study} most concatenation methods learn faster than the summation methods, but also is prone to early stopping because it starts to decline in performance.
+This also happens on \autoref{fig:GCF-NDCG-ablation-study-amazon-book} that use Amazon-book,but some concatenation methods still keeps learning unlike on Yelp2020.
+On \autoref{fig:GCF-NDCG-ablation-study-amazon-cell-sport} the methods perform differently compared to the Yelp2020 dataset.
+The summation methods train for a shorter amount of time compared to the Yelp2020 dataset.
+This could be due to the smaller datasets.
+Generally it can be seen that GCF and GCF-minus-sc performs better than the other methods used in the Amazon-Cell-Sport dataset.
```

Comment:
```suggestion
Generally it can be seen that GCF and GCF-minus-sc perform better than the other methods in \autoref{fig:GCF-NDCG-ablation-study-amazon-cell-sport}.
```